cache/active_user_cache.go

```diff
@@ -0,0 +1,45 @@
+package cache
+
+import (
+	"errors"
+	"github.com/vlpolak/swtgo/domain/entity"
+	"github.com/vlpolak/swtgo/domain/repository"
+	"sync"
+)
+
+type cachedUser struct {
+	*entity.User
+}
+
+type LocalCache struct {
+	mu             sync.RWMutex
```

Comment:
Recommendation: Naming does matter! In such cases we may use embedding:
```
type LocalCache struct {
	sync.RWMutex
	user           map[string]cachedUser
	userRepository repository.UserRepository
}
```
And using the proper naming, we may have clean human-readable code, like this:
```
	cache := &LocalCache{
		user:           make(map[string]cachedUser),
		userRepository: ur,
	}
	cache.Lock()		// each teammate who never seen that code before
	defer cache.Unlock()	// will understand what this code is used for
```